utilities/ghost-exporter/Converters.fs

```diff
@@ -2,10 +2,126 @@ module Converters
 
 open CMSSyntax
 open GhostSyntax
+open GhostSyntax.Sections
+open GhostSyntax.Cards
+
+let findTextStyle (markups: Markups.markup list) (markup: Markups.markup) : bool = 
+    List.exists (fun m -> m = markup) markups
+
+let convertMarkups (markups: Markups.markup list) : textStyle = 
+    {
+        bold = findTextStyle markups Markups.Strong // Strong seems like a direct translation for bold despite also having a bold markup?
+        italic = findTextStyle markups Markups.Emphasis // Emphasis seems like a direct translation with italics despite there being an italics markup?
+        underline = false
+    }
+
+// This needs to be implemented properly once frontend is ready
+let convertCallout (callout: Cards.callout) : text list = 
+    [{
+        data = Text callout.calloutText
+        style = {
+            bold = false
+            italic = false
+            underline = false
+        }
+    }]
+
+// This needs to be implemented properly once frontend is ready
+let convertToggle (toggle: Cards.toggle) : text list = 
```

Comment:
Theres a fair amount of duplication amongst these functions, can we construct a generic version of these functions and then make each of these functions just a partial application of the generic function: ie perhaps smth like:
```fs
let genericFunc (constructor: a -> Text) a = constructor a
let convertCallout = genericFunc Text
let convertToggle = genericFunc Text 
```